### DISCUSSION TITLE
non-ZIP ZIP files

### DISCUSSION THREAD

**Discussion by TBugReporter** [WinMerge/winmerge#2062] [Q&A]
How do I get WinMerge to unZIP ZIP files that aren't named *.ZIP?  (Other than renaming them, of course...)

**Comment by sdottaka** [comment]
Enable the "Detect archive type from file signature" checkbox in the Archive Support category of the Options window. That way you can extract files that don't have the zip extension.
![image](https://github.com/WinMerge/winmerge/assets/98126/173ecf03-111b-4986-8e83-b1b2ff45ab51)
